content/v2/csidriver/features/powerscale.md

```diff
@@ -1,570 +1,659 @@
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+    name: static-prov-pod
+spec:
+    containers:
+      - name: test
+        image: docker.io/centos:latest
+        command: [ "/bin/sleep", "3600" ]
+        volumeMounts:
+          - mountPath: "/data0"
+            name: pvol
+    volumes:
+      - name: pvol
+        persistentVolumeClaim:
+            claimName: isilonstaticpvc
+```
+
+5. After the pod becomes `Ready` and `Running`, you can start to use this pod and volume.
+
+## PVC Creation Feature
+
+Following yaml content can be used to create a PVC without referring any PV.
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: testvolume
+  namespace: default
+spec:
+  accessModes:
+  - ReadWriteMany
+  resources:
+        requests:
+          storage: 5Gi
+  storageClassName: isilon           
+```
+
+## Volume Snapshot Feature
+
+The CSI PowerScale driver version 2.0 and later supports managing v1 snapshots. 
+
+In order to use Volume Snapshots, ensure the following components have been deployed to your cluster:
+
+- Kubernetes Volume Snapshot CRDs
+- Volume Snapshot Controller
+
+> For general use, update the snapshot controller YAMLs with an appropriate namespace before installing. For
+> example, on a Vanilla Kubernetes cluster, update the namespace from default to kube-system before issuing the
+> kubectl create command.
+
+### Volume Snapshot Class
+
+During the installation of CSI PowerScale driver version 2.0 and higher, no default Volume Snapshot Class will get created.
+
+Following are the manifests for the Volume Snapshot Class:
+
+1. VolumeSnapshotClass 
+```yaml
+
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshotClass
+metadata:
+  name: "isilon-snapclass"
+driver: csi-isilon.dellemc.com
+#The deletionPolicy of a volume snapshot class can either be Retain or Delete
+#If the deletionPolicy is Delete, then the underlying storage snapshot is deleted along with the VolumeSnapshotContent object.
+#If the deletionPolicy is Retain, then both the underlying snapshot and VolumeSnapshotContent remain
+deletionPolicy: Delete
+parameters:
+  #IsiPath should match with respective storageClass IsiPath
+  IsiPath: "/ifs/data/csi"
+```
+
+The following is a sample manifest for creating a Volume Snapshot using the **v1** snapshot APIs; The following snippet assumes that the persistent volume claim name is testvolume.
+
+```yaml  
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: pvcsnap
+  namespace: default
+spec:
+  volumeSnapshotClassName: isilon-snapclass
+  source:
+    persistentVolumeClaimName: testvolume
+```
+
+Once the VolumeSnapshot has been successfully created by the CSI PowerScale driver, a VolumeSnapshotContent object is automatically created. Once the status of the VolumeSnapshot object has the _readyToUse_ field set to _true_ , it is available for use.
+
+Following is the relevant section of VolumeSnapshot object status:

```

Comment:
Insert "The" before "Following". Remove initial uppercase on "Following".

Reply:
Addressed the comment